ada_2_fhir-r4/fhir/2_fhir_fhir_include.xsl

```diff
@@ -93,7 +95,6 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         <nm:map ada="lichaamslengte" resource="Observation" profile="nl-core-BodyHeight"/>
         <nm:map ada="lichaamstemperatuur" resource="Observation" profile="nl-core-BodyTemperature"/>
         <nm:map ada="lichaamsgewicht" resource="Observation" profile="nl-core-BodyWeight"/>
-        <nm:map ada="medicatie_contra_indicatie" resource="Flag" profile="cio-MedicationContraIndication"/>
         <nm:map ada="medicatie_contra_indicatie" resource="Flag" profile="nl-core-MedicationContraIndication"/>
```

Comment:
Waarom geen mapping meer naar cio-MedicationContraIndication, die maak je nu toch juist?
En wat is de mapping naar nl-core-MedicationContraIndication, bestaat die überhaupt in zibs2020?

Reply:
Helder, dank!